modules/i18n/src/translations/en-US/portals/remote-user-stores.ts

```diff
@@ -0,0 +1,245 @@
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+/* eslint-disable max-len */
+
+import { RemoteUserStoresNS } from "../../../models/namespaces/remote-user-stores-ns";
+
+export const remoteUserStores: RemoteUserStoresNS = {
+    form: {
+        fields: {
+            accessType: {
+                label: "Access Type",
+                placeholder: "Select access type",
+                validation: {
+                    required: "Access type is required"
+                }
+            },
+            attributes: {
+                validation: {
+                    required: "Attribute mapping is required"
+                }
+            },
+            connectedUserStoreType: {
+                label: "Remote User Store Type",
+                placeholder: "Select the type of remote user store",
+                validation: {
+                    required: "User store type is required"
+                }
+            },
+            description: {
+                label: "Description",
+                placeholder: "Describe the purpose or scope of this user store"
+            },
+            groupIdMapping: {
+                helperText: "Specify the attribute from the user store that stores the group ID.",
+                label: "Group ID",
+                placeholder: "Ex: groupId",
+                validation: {
+                    required: "Group ID mapping is required when read groups is enabled"
+                }
+            },
+            groupnameMapping: {
+                helperText: "Specify the attribute from the user store that stores the group name.",
+                label: "Group Name",
+                placeholder: "Ex: groupName",
+                validation: {
+                    required: "Group name mapping is required when read groups is enabled"
+                }
+            },
+            name: {
+                label: "User Store Name",
+                placeholder: "ex: MY USER STORE",
+                validation: {
+                    required: "User store name is required"
+                }
+            },
+            readGroups: {
+                helperText: "Enable this option to retrieve group information from the user store.",
+                label: "Read Groups"
+            },
+            userIdMapping: {
+                helperText: "Specify the attribute from the user store that represents the unique ID for the user.",
+                label: "User ID Mapping",
+                placeholder: "Ex: uid",
+                validation: {
+                    required: "User ID mapping is required"
+                }
+            },
+            usernameMapping: {
+                helperText: "Specify the attribute from the user store that represents the user's primary identifier.",
+                label: "Username Mapping",
+                placeholder: "Ex: un",
```

Comment:
let's use e.g. instead of ex:

Reply:
Updated 70f200f14255f037a46096205acd8f666919bb76